### PR TITLE
Refer to Git instead of Subversion in documentation

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

**Comment by adiroiban** [comment]
Thanks for the contribution.

PR are not yet a accepted. See #61 

You can host the code in github but please create a ticket inside Trac and ask for a review of that ticket

**Comment by adiroiban** [comment]
Ok. Got it https://twistedmatrix.com/trac/ticket/8335

**Comment by rodrigc** [comment]
@glyph @adiroiban @hawkowl @mithrandi
I followed https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch

Maybe the text there needs to be changed so that a ticket is created before the pull request is done?
That way the ticket can be referred to in the pull request itself.

File: docs/core/development/policy/coding-standard.rst [hunk 1]
```diff
@@ -647,24 +647,24 @@ If you can't summarize your changes in one short line, this is probably a sign t
...
-
-- Do not set the ``svn:ignore`` property on any file or directory.
-  What you wish to ignore, others may wish to examine.
-  What others may wish you ignore, *you* may wish you examine.
-  ``svn:ignore`` will affect everyone who uses the repository, and so it is not the right mechanism to express personal preferences.
-
-  If you wish to ignore certain files use the ``global-ignores`` feature of ``~/.subversion/config``, for example:
-
-  .. code-block:: console
+If you wish to ignore certain files, create a ``~/.gitconfig`` file.
```
Comment (hawkowl) [inline docs/core/development/policy/coding-standard.rst]: We do have a `.gitignore`, which is reasonable to add things to.

File: docs/core/development/policy/coding-standard.rst [hunk 2]
```diff
@@ -637,7 +637,7 @@ Therefore, it should be short (aim for < 80 characters) and descriptive -- and m
 The rest of the e-mail should be separated with *hard line breaks* into short lines (< 70 characters).
 This is free-format, so you can do whatever you like here.
 
-Commit messages should be about *what*, not *how*: we can get how from SVN diff.
+Commit messages should be about *what*, not *how*: we can get how from git diff.
```
Comment (hawkowl) [inline docs/core/development/policy/coding-standard.rst]: git -> Git

File: docs/core/development/policy/code-dev.rst [hunk 3]
```diff
@@ -233,7 +218,7 @@ Committing and Post-commit Hooks
 
 
 
-Twisted uses a customized `trac-post-commit-hook <http://bazaar.launchpad.net/~exarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook>`_ to enable ticket updates based on svn commit
+Twisted uses a customized `trac-post-commit-hook <http://bazaar.launchpad.net/~exarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook>`_ to enable ticket updates based on git commit
```
Comment (hawkowl) [inline docs/core/development/policy/code-dev.rst]: This should be changed to say "Twisted's Trac installation is notified when the Git repository changes, and will update the ticket depending on the Git commit logs." (We don't use trac-post-commit-hook anymore)

File: docs/core/development/policy/code-dev.rst [hunk 4]
```diff
@@ -3,7 +3,7 @@
 :LastChangedRevision: $LastChangedRevision$
 :LastChangedBy: $LastChangedBy$
 
-Working from Twisted's Subversion repository
+Working from Twisted's code repository
```
Comment (hawkowl) [inline docs/core/development/policy/code-dev.rst:6]: I feel like this should say Git rather than code, "code repository" sounds a bit eh to me.

**Comment by rodrigc** [comment]
OK, I incorporated all your changes.

**Comment by hawkowl** [comment]
Thanks, I've pulled your changes onto the branch listed on https://twistedmatrix.com/trac/ticket/8335.

**Comment by rodrigc** [comment]
Nice!  Am I the first person to have successfully submitted to Twisted via GitHub pull request: https://github.com/twisted/twisted/commit/7593887c815e67a7fe5705b5be804e72a65a9a15 ?

**Comment by hawkowl** [comment]
@rodrigc yes ;)

**Pull request closed without merging**